payloads/library/prank/lockpc/payload.txt

```diff
@@ -0,0 +1,68 @@
+#
+#
+# Title:             PC Lock Fun
+# Original Authors:            The10FpsGuy and Mrhut10
+#                    Big thanks to Mrhut10 for helping to put a loop in it :D
+# Modified:	     FraterVI
+# Category:          Fun
+# Target:.           Windows, Linux, Mac
+# Description:	     Takes original prank payload and adds OS detection so
+#		     it can be run on any(mostly any) OS.
+#
+#options
+mount -o /dev/nandf /root/udisk

```

Comment:
Please don't do this - nandf is already mounted to /root/udisk. If for some reason it wasn't, please use the following command: `udisk mount`